cypress/integration/forgotPassword.spec.js

```diff
@@ -0,0 +1,35 @@
+const autoRecord = require('cypress-autorecord');
+
+describe('The Forgot Password Page', () => {
+  autoRecord();
+
+  beforeEach(() => {
+    cy.visit('/forgot-password')
+  })
+
+  describe('Form submit with a valid email', () => {
+    it('shows validation error if submitted empty email', function () {
+      cy.get('form').submit()
+      cy.url().should('include', '/forgot-password')
+      cy
+        .get('.validation-error')
+        .should(($e) => {
+          expect($e).to.have.length(1)
+          expect($e).to.contain('Required')
+        })
+    })
```

Comment:
@rafayet-monon  Should this be in another `describe`? And how about moving `if submitted empty email` to a context?

```suggestion
describe('Form submit with an INVALID email', () => {
  context('given empty email', () => {
    it('shows validation error', () => {
      cy.get('form').submit()
      cy.url().should('include', '/forgot-password')
      cy
        .get('.validation-error')
        .should(($e) => {
          expect($e).to.have.length(1)
          expect($e).to.contain('Required')
        })
    })
  })
})

describe('Form submit with a valid email', () => {
```